apps/test/unit/templates/curriculumCatalog/CurriculumCatalogTest.jsx

```diff
@@ -671,15 +672,62 @@ describe('CurriculumCatalog', () => {
         // Get the Similar Recommended Curriculum for the current test curriculum
         const recommendedSimilarCurriculum = getSimilarRecommendations(
           FULL_TEST_COURSES,
-          currCurriculum.key
+          currCurriculum.key,
+          null
         )[0];
 
         // Open expanded card of the current test curriculum
         fireEvent.click(quickViewButtons[i]);
         screen.getByText(currCurriculum.description);
 
         // Check that the recommended similar curriculum's image and link are present on the current test curriculum's expanded card.
-        //Image's alt text is the curriculum's display name.
+        // Image's alt text is the curriculum's display name.
+        screen.getByAltText(recommendedSimilarCurriculum.display_name);
```

Comment:
prefacing this with the fact that I'm learning RTL, but can you use `screen.getByRole('img', {name: recommendedSimilarCurriculum.display_name})` here instead? Also, my understanding was that `ByRole` was preferable but that might be wrong!

Reply:
I'm not sure it will work becasue I think it will only find it by name if the image tag has no 'alt' tag but the image needs an 'alt' tag to be accessible. I'm also learning RTL so I might be wrong!

Reply:
oh interesting! good to know!